rb/lib/selenium/webdriver/common/error.rb

```diff
@@ -37,7 +38,15 @@ def self.for_error(error)
       SUPPORT_MSG = 'For documentation on this error, please visit:'
       ERROR_URL = 'https://www.selenium.dev/documentation/webdriver/troubleshooting/errors'
 
-      class WebDriverError < StandardError; end
+      class WebDriverError < StandardError
+        def backtrace_locations
+          Thread.current.backtrace_locations
```

Comment:
This can change at any time, you need to save it. Same with `$ERROR_INFO`.

Reply:
Thank you for the review! I saved it now in a variable and added an attribute writer and type checks, the tests are still passing